views/partials/footer.hbs

```diff
@@ -33,7 +33,10 @@
             </ul>
           </div>
           <div class="row legal">
-            <p class="half copyright-info">{{{fluentFormat req.supportedLocales "copyright-info"}}}</p>
```

Comment:
Do we need to delete the "copyright-info" string from the .ftl file?

Reply:
Yes, we should